lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -678,15 +669,15 @@ public void draw(@NonNull Canvas canvas) {
 
       if (drawTexture) {
         // If we should use a texture, draw it instead of text
-        canvas.drawBitmap(expandedTitleTexture, x, y + ascent, texturePaint);
```

Comment:
Since `y` marks top of the text to draw, no need to add (actually subtract, see why it is a bit confusing?) the `ascent`